doc/sphinx/proofs/writing-proofs/reasoning-inductives.rst

```diff
@@ -1065,6 +1081,35 @@ Generation of induction principles with ``Scheme``
   The type of `even_odd` shares the same premises but the conclusion is
   `forall n : nat, even n -> P0 n`.
 
+.. _scheme_example:
+
+   .. example:: `Scheme` commands with various :n:`@scheme_type`\s
+
+      Let us demonstrate the difference between the Scheme commands.
+
+      .. coqtop:: in
```

Comment:
If you don't include "all" or "out" here, there won't be any output--making the "Abouts" pointless.